### ISSUE TITLE
LASSIS with ncharge and nspin error

### ISSUE THREAD

**Issue by valay1** [MatthewRHermes/mrh#84]
When trying to do a PES scan for bisdiazine with (4,4)^(2,2)^(4,4) active space, I get the following error when using LASSIS with ncharge and nspin.
Error: 
  3 Traceback (most recent call last):
  4   File "/project/lgagliardi/valayagarawal/lasscf/pes_lassis/bisdiazene/c2h4n4_631g_las3_fscan.py", line 33, in <module>
  5     lsi = LASSIS (las,ncharge=2,nspin=2).run ()
  6   File "/project/lgagliardi/shared/Apps/pyscf/pyscf/lib/misc.py", line 612, in run
  7     self.kernel(*args)
  8   File "/project/lgagliardi/valayagarawal/Apps/mrh-cpu/mrh/my_pyscf/lassi/lassis.py", line 456, in kernel
  9     self.converged = self.prepare_states_(ncharge=ncharge, nspin=nspin,
 10   File "/project/lgagliardi/valayagarawal/Apps/mrh-cpu/mrh/my_pyscf/lassi/lassis.py", line 471, in prepare_states_
 11     self.converged, las = self.prepare_states (ncharge=ncharge, nspin=nspin,
 12   File "/project/lgagliardi/valayagarawal/Apps/mrh-cpu/mrh/my_pyscf/lassi/lassis.py", line 59, in prepare_states
 13     converged, las2.ci, las2.e_states = single_excitations_ci (
 14   File "/project/lgagliardi/valayagarawal/Apps/mrh-cpu/mrh/my_pyscf/lassi/lassis.py", line 139, in single_excitations_ci
 15     assert (np.all (lroots_i==lroots[:,i])), "{} {} {} {}".format (
 16 AssertionError: 1 [2, 2, 1] 7 [2 2 2]

This works when I do not specify the ncharge and npsin.

**Comment by MatthewRHermes** [comment]
Keep an eye on https://github.com/MatthewRHermes/mrh/actions/runs/8668744028/job/23774333702. If it passes, pull dev and try again.

**Comment by MatthewRHermes** [comment]
Explanation: I was really lazy and set every element of the `lroots` table to, at minimum, the `ncharge` integer provided by the caller, because it's never really used for the spectator fragment anyway. But then, later on, I went and wrote a sanity-checking assert that looked at the entire row and oops, third entry's wrong! So I fixed it, but I put the fix in an "if" block that it only enters for the automatic `ncharge` determination. So I had to delete a tab.
